lib/Entity/VideosRequestParameters.php

```diff
@@ -34,6 +34,8 @@
  * @method VideosRequestParameters setIncludeSubChannels(bool $includeSubChannels)
  * @method VideosRequestParameters setMetadataSetKey(string $metadataSetKey)
  * @method string                  getMetadataSetKey()
+ * @method string                  getLocale()
```

Comment:
That is only needed for IDE autocomplete. As there is no IDE on this planet which understands such magic here :)
https://github.com/MovingImage24/VMProApiClient/blob/master/lib/Util/AccessorTrait.php#L23